securedrop/journalist_app/admin.py

```diff
@@ -85,10 +85,9 @@ def manage_config() -> Union[str, werkzeug.Response]:
                     gettext("Unable to process the image file. Please try another one."),
                     "logo-error",
                 )
-            finally:
-                return redirect(url_for("admin.manage_config") + "#config-logoimage")
+            return redirect(url_for("admin.manage_config") + "#config-logoimage")
         else:
-            for field, errors in list(logo_form.errors.items()):
+            for errors in list(logo_form.errors.values()):
```

Comment:
Can we remove the `list()` calls while we're here, since we're iterating over `errors` anyway?  (We don't bother with with `list()` at all in `source_app.main`.)